apps/web/src/app/(dashboard)/admin/page.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Archive,
+  File,
+  FileX2,
+  LucideIcon,
+  User as LucideUser,
```

Comment:
I think it would be better named as `User as UserIcon`

iirc lucide should be exporting `UserIcon` anyway but I assume we're doing something weird so it doesn't work or something